xradar/io/backends/cfradial1.py

```diff
@@ -94,9 +94,28 @@ def _get_required_root_dataset(ds, optional=True):
     return root
 
 
-def _get_sweep_groups(root, sweep=None, first_dim="time"):
+def _get_sweep_groups(
+    root, sweep=None, first_dim="time", optional=True, site_coords=False
```

Comment:
Can we set the first_dim to auto, and the site_coords to true by default? Both of these would make dealing with this easier.

Reply:
@mgrover1 Yes, but then we would need our own dedicated cfradial1 writer to swap dims backs and drop vars/coords out.

Should I add this functionality?

Reply:
Yeah - I think this would help!

Reply:
It's similar to how one could use dataset.to_netcdf(), but it does not necessarily conform to the additional metadata standards as a cfradial file (dataset.to_cfradial1...)

Reply:
@mgrover1 I've added a prototype which we can build on. Need to fix/add tests, but you might already have an early look at this.